npm/mount-utils/src/index.ts

```diff
@@ -39,6 +39,10 @@ export interface StyleOptions {
 
 export const ROOT_SELECTOR = '[data-cy-root]'
 
+/**
+ * Gets the root element used to mount the component.
+ * @returns {HTMLElement | null} The root element
+ */
```

Comment:
Since we've got the type definition, I don't think we need this particular annotation? It's also incorrect - this method either returns a `HTMLDivElement` or it throws - there's no situation where it returns `null`.